src/Repository/ReaderRepository.php

```diff
@@ -2,26 +2,87 @@
 
 namespace BookReviews\Repository;
 
+use \PDO;
+
 /**
  * Class ReaderRepository
  * @package BookReview\Repository
  */
-class ReaderRepository extends DatabaseConnection
+class ReaderRepository implements Repository
 {
     /**
-     * @param string $username
-     * @param string $password
+     * @var PDO
      */
-    public function insertIntoDatabase($username, $password)
+    private $conn;
+
+    public function __construct()
+    {
+        $database = new DatabaseConnectionBuilder();
+        $this->conn = $database->connectToDatabase();
+    }
+
+    public function getId()
+    {
+        // TODO: Implement getId() method.
+    }
+
+    public function add()
+    {
+        // TODO: Implement add() method.
+    }
+
+    public function update()
+    {
+        // TODO: Implement update() method.
+    }
+
+    public function delete()
     {
-         die();
+        // TODO: Implement delete() method.
     }
     /**
+     * The function returns  true if the result of the query returns an array
+     * and the password matches the one in the array.
      * @param string $username
      * @param string $password
+     * @return boolean
      */
     public function findInDatabase($username, $password)
     {
-        die();
+        $query = "Select * from readers where username= \"{$username}\"";
```

Comment:
Mysql keywords should be all uppercase.
Never use `{$varname}`, instead use the method [sprintf](http://php.net/manual/ro/function.sprintf.php).

For any sql queries where you need to have a variable value inside them, use parameter binding with [bindValue](http://php.net/manual/en/pdostatement.bindvalue.php) and [bindParam](http://php.net/manual/en/pdostatement.bindparam.php)